ignitions/kubelet/resources/services/kubelet.service

```diff
@@ -32,6 +32,7 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
   --minimum-container-ttl-duration=6m0s \
   --client-ca-file=/etc/kubernetes/ca.crt \
   --cluster-domain=cluster.local \
+  --authentication-token-webhook=true \
```

Comment:
What is this for ?

Reply:
Have removed and passed it from kubelet extra flag variable